python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -834,6 +844,9 @@ router bgp {{ router_bgp.as }}
    !
    address-family ipv4
 {%         for peer_group in router_bgp.address_family_ipv4.peer_groups | arista.avd.natural_sort('name') %}
+{%             if peer_group.additional_paths.receive is arista.avd.defined %}
```

Comment:
in schema you added the additional path support for neighbor and bgp additional path but in template you just added for peer group.
Also please confirm that should we add it as new feature for all other AFI. I think we should just update for existing one